plugins/pencilblue/controllers/article.js

```diff
@@ -33,11 +33,9 @@ Article.prototype.render = function(cb) {
 	var custUrl = this.pathVars.customUrl;
 
 	//check for object ID as the custom URL
-	var doRedirect = false;
-	var where      = null;
+	var where  = null;
 	try {
```

Comment:
This is a lot like the portfolio bug we fixed.  We need to do the validation check on the ```custUrl``` variable instead of the try catch block.  That will allow us to determine if it is an article ID or custom URL.